### PR TITLE
Added note about custom domain in fetched URL for Session Replay integrations

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Adds a note about including a subdomain in the function call.

### Motivation
[Suggestion](https://github.com/DataDog/documentation/pull/17889#discussion_r1178776673) from the mobile team.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by rtrieu** [comment]
Hi @acorretti, I would appreciate a review of this addition, just to be sure I understood your suggestion correctly. Please feel free to rephrase as a suggestion and I can accept. Thanks again for the suggestion!

**Comment by rtrieu** [comment]
Hi @acorretti can you review this PR? Thanks!

File: content/en/real_user_monitoring/guide/connect-session-replay-to-your-third-party-tools.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ You may want to connect a third-party tool with Session Replay for a more compre
 
 To fetch the URL for the current user session's recording, use the following snippet, depending on the installation method you used to set up RUM:
 
+**Note**: Although providing a value for `subdomain` when fetching the user session's recording URL is optional, you should provide it if you're accessing Datadog through a custom subdomain and want to see the custom domain in the URL that gets returned.
```
Comment (buraizu) [inline content/en/real_user_monitoring/guide/connect-session-replay-to-your-third-party-tools.md:26]:
```suggestion
**Note**: Providing a value for `subdomain` when fetching the user session's recording URL is optional, but must be provided if you're accessing Datadog through a custom subdomain and want to see the custom domain in the URL that gets returned.
```

Comment (buraizu) [reply content/en/real_user_monitoring/guide/connect-session-replay-to-your-third-party-tools.md:26]: [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) is the source of truth for determining the meanings of words like `should` and `must` -- I'm guessing in this case that the correct word is `must`, but feel free to ignore this suggestion if that's incorrect.

**Review by buraizu — APPROVED** [review]
Approving with some minor suggestions!

**Review by acorretti — APPROVED** [review]
I'm so sorry to have missed this PR, @rtrieu !
Looks good, although count my +1 for @buraizu 's suggestion

File: content/en/real_user_monitoring/guide/connect-session-replay-to-your-third-party-tools.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/guide/connect-session-replay-to-your-third-party-tools.md]: Thank you @buraizu for the suggestion. It is now much clearer!

**Pull request merged**